### PR TITLE
Now that most of the code is done, let's add CI to help with further development that won't exist

### PR DESCRIPTION
**Pull request by alexkar598**

### CODE REVIEW

**Comment by alexkar598** [comment]
hold on, git fucky happened

**Comment by alexkar598** [comment]
Git has been un fuckied

**Comment by ZeWaka** [comment]
haha rekt

File: .github/workflows/rust.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+env:
+  CARGO_TERM_COLOR: always
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
```
Comment (ZeWaka) [inline .github/workflows/rust.yml:17]: `>no clippy`

Comment (alexkar598) [reply .github/workflows/rust.yml:17]: Who's clippy

**Review by ZeWaka — APPROVED** [review]

**Comment by alexkar598** [comment]
am gud @ cooding

**Pull request merged**